diff -Nru phpMyAdmin-2.6.2.orig/config.inc.php phpMyAdmin-2.6.2/config.inc.php
--- phpMyAdmin-2.6.2.orig/config.inc.php	2005-03-16 18:22:08 +0100
+++ phpMyAdmin-2.6.2/config.inc.php	2005-05-05 12:11:32 +0200
@@ -44,13 +44,13 @@
  * You should use this if and ONLY if the PmaAbsoluteUri auto-detection
  * works perfectly.
  */
-$cfg['PmaAbsoluteUri_DisableWarning'] = FALSE;
+$cfg['PmaAbsoluteUri_DisableWarning'] = TRUE;
 
 /**
  * Disable the default warning that is displayed on the DB Details Structure page if
  * any of the required Tables for the relationfeatures could not be found
  */
-$cfg['PmaNoRelation_DisableWarning']  = FALSE;
+$cfg['PmaNoRelation_DisableWarning']  = TRUE;
 
 /**
  * The 'cookie' auth_type uses blowfish algorithm to encrypt the password. If
@@ -59,6 +59,7 @@
  * characters.
  */
 $cfg['blowfish_secret'] = '';
+include('/etc/phpmyadmin/blowfish_secret.inc.php');
 
 /**
  * Server(s) configuration
@@ -70,7 +71,7 @@
 $cfg['Servers'][$i]['host']          = 'localhost'; // MySQL hostname or IP address
 $cfg['Servers'][$i]['port']          = '';          // MySQL port - leave blank for default port
 $cfg['Servers'][$i]['socket']        = '';          // Path to the socket - leave blank for default socket
-$cfg['Servers'][$i]['connect_type']  = 'tcp';       // How to connect to MySQL server ('tcp' or 'socket')
+$cfg['Servers'][$i]['connect_type']  = 'socket';    // How to connect to MySQL server ('tcp' or 'socket')
 $cfg['Servers'][$i]['extension']     = 'mysql';     // The php MySQL extension to use ('mysql' or 'mysqli')
 $cfg['Servers'][$i]['compress']      = FALSE;       // Use compressed protocol for the MySQL connection
                                                     // (requires PHP >= 4.3.0)
@@ -81,7 +82,7 @@
                                                     // The controluser is also
                                                     // used for all relational
                                                     // features (pmadb)
-$cfg['Servers'][$i]['auth_type']     = 'config';    // Authentication method (config, http or cookie based)?
+$cfg['Servers'][$i]['auth_type']     = 'cookie';    // Authentication method (config, http or cookie based)?
 $cfg['Servers'][$i]['user']          = 'root';      // MySQL user
 $cfg['Servers'][$i]['password']      = '';          // MySQL password (only needed
                                                     // with 'config' auth_type)
@@ -838,6 +839,13 @@
  */
 set_magic_quotes_runtime(0);
 
+
+/**
+ * Include Debian local configuration
+ */
+include('/etc/phpmyadmin/config.inc.php');
+
+
 /**
  * File Revision - do not change either!
  */
