diff -ru phpMyAdmin-2.6.1-rc1.orig/config.inc.php phpMyAdmin-2.6.1-rc1/config.inc.php
--- phpMyAdmin-2.6.1-rc1.orig/config.inc.php	2004-11-03 14:45:25 +0100
+++ phpMyAdmin-2.6.1-rc1/config.inc.php	2004-12-13 19:52:32 +0100
@@ -44,13 +44,13 @@
  * You should use this if and ONLY if the PmaAbsoluteUri auto-detection
  * works perfectly.
  */
-$cfg['PmaAbsoluteUri_DisableWarning'] = FALSE;
+$cfg['PmaAbsoluteUri_DisableWarning'] = TRUE;
 
 /**
  * Disable the default warning that is displayed on the DB Details Structure page if
  * any of the required Tables for the relationfeatures could not be found
  */
-$cfg['PmaNoRelation_DisableWarning']  = FALSE;
+$cfg['PmaNoRelation_DisableWarning']  = TRUE;
 
 /**
  * The 'cookie' auth_type uses blowfish algorithm to encrypt the password. If
@@ -58,6 +58,7 @@
  * passphrase that will be used by blowfish.
  */
 $cfg['blowfish_secret'] = '';
+include('/etc/phpmyadmin/blowfish_secret.inc.php');
 
 /**
  * Server(s) configuration
@@ -69,7 +70,7 @@
 $cfg['Servers'][$i]['host']          = 'localhost'; // MySQL hostname or IP address
 $cfg['Servers'][$i]['port']          = '';          // MySQL port - leave blank for default port
 $cfg['Servers'][$i]['socket']        = '';          // Path to the socket - leave blank for default socket
-$cfg['Servers'][$i]['connect_type']  = 'tcp';       // How to connect to MySQL server ('tcp' or 'socket')
+$cfg['Servers'][$i]['connect_type']  = 'socket';    // How to connect to MySQL server ('tcp' or 'socket')
 $cfg['Servers'][$i]['extension']     = 'mysql';     // The php MySQL extension to use ('mysql' or 'mysqli')
 $cfg['Servers'][$i]['compress']      = FALSE;       // Use compressed protocol for the MySQL connection
                                                     // (requires PHP >= 4.3.0)
@@ -80,7 +81,7 @@
                                                     // The controluser is also
                                                     // used for all relational
                                                     // features (pmadb)
-$cfg['Servers'][$i]['auth_type']     = 'config';    // Authentication method (config, http or cookie based)?
+$cfg['Servers'][$i]['auth_type']     = 'cookie';    // Authentication method (config, http or cookie based)?
 $cfg['Servers'][$i]['user']          = 'root';      // MySQL user
 $cfg['Servers'][$i]['password']      = '';          // MySQL password (only needed
                                                     // with 'config' auth_type)
@@ -825,6 +826,33 @@
  */
 set_magic_quotes_runtime(0);
 
+
+/**
+ * Include Debian local configuration
+ */
+include('/etc/phpmyadmin/config.inc.php');
+
+
+/**
+ * Load extension if not loaded yet
+ */
+$extensions = array();
+$servers_cnt = count($cfg['Servers']);
+for ($i = 1; $i <= $servers_cnt; $i++) {
+    if (isset($cfg['Servers'][$i]) 
+      && !empty($cfg['Servers'][$i]['host']) 
+      && isset($cfg['Servers'][$i]['extension'])) {
+	$extensions[$cfg['Servers'][$i]['extension']] = 1;
+    }
+}
+if (isset($extensions['mysql']) && !@extension_loaded('mysql') && !@get_cfg_var('safe_mode')) {
+    @dl('mysql.so');
+}
+if (isset($extensions['mysqli']) && !@extension_loaded('mysqli') && !@get_cfg_var('safe_mode')) {
+    @dl('mysqli.so');
+}
+
+
 /**
  * File Revision - do not change either!
  */
