diff -Nru phpMyAdmin-2.6.1-pl3.orig/libraries/defines.lib.php phpMyAdmin-2.6.1-pl3/libraries/defines.lib.php
--- phpMyAdmin-2.6.1-pl3.orig/libraries/defines.lib.php	2005-02-24 20:30:31 +0100
+++ phpMyAdmin-2.6.1-pl3/libraries/defines.lib.php	2005-03-07 12:24:11 +0100
@@ -73,20 +73,7 @@
 function PMA_dl($module) {
     if (!isset($GLOBALS['PMA_dl_allowed'])) {
         if (!@ini_get('safe_mode') && @ini_get('enable_dl') && @function_exists('dl')) {
-            ob_start();
-            phpinfo(INFO_GENERAL); /* Only general info */
-            $a = strip_tags(ob_get_contents());
-            ob_end_clean();
-            /* Get GD version string from phpinfo output */
-            if (preg_match('@Thread Safety[[:space:]]*enabled@', $a)) {
-                if (preg_match('@Server API[[:space:]]*\(CGI\|CLI\)@', $a)) {
-                    $GLOBALS['PMA_dl_allowed'] = TRUE;
-                } else {
-                    $GLOBALS['PMA_dl_allowed'] = FALSE;
-                }
-            } else {
-                $GLOBALS['PMA_dl_allowed'] = TRUE;
-            }
+            $GLOBALS['PMA_dl_allowed'] = TRUE;
         } else {
             $GLOBALS['PMA_dl_allowed'] = FALSE;
         }
