diff -Nru phpMyAdmin-2.8.0.1.orig/scripts/setup.php phpMyAdmin-2.8.0.1/scripts/setup.php
--- phpMyAdmin-2.8.0.1.orig/scripts/setup.php	2006-02-06 12:01:48.000000000 +0100
+++ phpMyAdmin-2.8.0.1/scripts/setup.php	2006-03-15 12:39:56.000000000 +0100
@@ -1220,10 +1220,6 @@
                 message('error', 'Empty hostname!');
                 $err = TRUE;
             }
-            if ($new_server['connect_type'] == 'socket' && empty($new_server['socket'])) {
-                message('error', 'Empty socket with socket connection seleted!');
-                $err = TRUE;
-            }
             if ($new_server['auth_type'] == 'config' && empty($new_server['user'])) {
                 message('error', 'Empty username while using config authentication method!');
                 $err = TRUE;
