diff -u phpmyadmin-2.2.2-2.2.3-dev-20011217.orig/config.inc.php phpmyadmin-2.2.2-2.2.3-dev-20011217/config.inc.php
--- phpmyadmin-2.2.2-2.2.3-dev-20011217.orig/config.inc.php	Mon Dec 17 16:15:23 2001
+++ phpmyadmin-2.2.2-2.2.3-dev-20011217/config.inc.php	Tue Dec 18 12:41:13 2001
@@ -41,10 +41,10 @@
 $cfgServers[1]['host']          = 'localhost'; // MySQL hostname
 $cfgServers[1]['port']          = '';          // MySQL port - leave blank for default port
 $cfgServers[1]['socket']        = '';          // Path to the socket - leave blank for default socket
-$cfgServers[1]['connect_type']  = 'tcp';       // How to connect to MySQL server ('tcp' or 'socket')
+$cfgServers[1]['connect_type']  = 'socket';    // How to connect to MySQL server ('tcp' or 'socket')
 $cfgServers[1]['stduser']       = '';          // MySQL standard user settings (this user must have read-only
 $cfgServers[1]['stdpass']       = '';          //   access to the "mysql/user" and "mysql/db" tables)
-$cfgServers[1]['auth_type']     = 'config';     // Authentication method (config, http or cookie based)?
+$cfgServers[1]['auth_type']     = 'cookie';    // Authentication method (config, http or cookie based)?
 $cfgServers[1]['user']          = 'root';      // MySQL user
 $cfgServers[1]['password']      = '';          // MySQL password (only needed with 'config' auth)
 $cfgServers[1]['only_db']       = '';          // If set to a db-name, only this db is displayed at left frame
@@ -250,4 +251,11 @@
  * Unset magic_quotes_runtime - do not change!
  */
 set_magic_quotes_runtime(0);
+
+
+// Load extension if not loaded yet
+if (!@extension_loaded("MySQL") && !@extension_loaded("mysql") && !@get_cfg_var('safe_mode')) {
+    @dl("mysql.so");
+}
+
 ?>
